src/Sarif.Sarifer/BackgroundAnalysisService.cs

```diff
@@ -24,13 +24,24 @@ internal class BackgroundAnalysisService : IBackgroundAnalysisService
 #pragma warning restore CS0649
 
         /// <inheritdoc/>
-        public async Task StartAnalysisAsync(string path, string text)
+        public async Task AnalyzeAsync(string path, string text)
         {
-            List<Task> tasks = new List<Task>(analyzers.Count());
+            var tasks = new List<Task>(analyzers.Count());
```

Comment:
should we remove or use the same idea as the method below?

Reply:
I should have used that idea. Thanks.

---
In reply to: [531726321](https://github.com/microsoft/sarif-visualstudio-extension/pull/285#discussion_r531726321) [](ancestors = 531726321)